src/Tasks/Common/ConflictResolution/ResolvePackageFileConflicts.cs

```diff
@@ -262,7 +262,9 @@ private ITaskItem[] RemoveConflicts(ITaskItem[] original, ICollection<ITaskItem>
             var result = new ITaskItem[original.Length - conflicts.Count];
             int index = 0;
 
-            foreach(var originalItem in original)
+            var originalConflicts = original.Where(o => conflicts.Contains(o)).ToArray();
```

Comment:
This appears to be unused.

Reply:
Oops! That was debugging code that got left over.  I've fixed it.